packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -835,21 +835,19 @@ export class EntityService {
         );
       }
 
-      for (const relatedEntityField of relatedEntityFields) {
-        await this.deleteField(
-          { where: { id: relatedEntityField.id } },
-          user,
-          fieldStrategy
-        );
-      }
-
       try {
         await this.moduleService.deleteDefaultModuleForEntity(
           entity.resourceId,
           entity.id,
           user
         );
       } catch (error) {
+        const amplicationError = error as AmplicationError;
+        if (
+          amplicationError.cause === "dtoInUse" ||
+          amplicationError.cause === "ActionDtoInUse"
+        )
+          throw new AmplicationError(error);
```

Comment:
The comments and log below no longer make sense (lines 851-856).

Reply:
Why? In case the error is thrown for another reason, the process will and should continue @mulygottlieb